packages/dso-toolkit/components/Componenten/search-bar/search-bar.config.yml

```diff
@@ -35,10 +35,6 @@ variants:
       __title: Met waarde
       value: Waarde
       clearButton: true
-  - name: invalid
```

Comment:
Dit blijft wel raar.. Het zit niet in next, het zit niet in master. Maar de commit die deze change realiseert zit wel in next maar niet in master, maar de oorsprong van de change zit alleen in master en niet in next

Reply:
Zie comment over cherry picks